package qindexdb;

message LogRecord {
    required uint64 seq_no = 1;
    required string key = 2;
    required string value = 3;
}

message RecordLocation {
    required uint64 log_file_no = 1;
    required int64 offset = 2;
    required uint64 record_size = 3;
}
